docs/generate_py_docs.py

```diff
@@ -139,11 +139,20 @@ def skip_or_replace_unparseable_directives(rst_: str) -> str:
 
 
 def resolve_includes(rst_: str, docs_source_path_: str) -> str:
-    matches = re.findall(r'(.*)\.\. include:: (.*)', rst_)
-    for indent_, match in matches:
+    matches = re.findall(r'(.*)\.\. include:: (.*)\n(.* :(.*): ([0-9]*)\n)?(.* :(.*): ([0-9]*)\n)?', rst_)
```

Comment:
Wowow this is pretty complicated, pls include (in a code comment) an example what text is matched and what is extracted of that